arangod/Aql/RestAqlHandler.cpp

```diff
@@ -651,31 +651,10 @@ auto AqlExecuteCall::fromVelocyPack(VPackSlice const slice) -> ResultT<AqlExecut
 
 // handle for useQuery
 RestStatus RestAqlHandler::handleUseQuery(std::string const& operation,
-                                          VPackSlice const querySlice) {
+                                          VPackSlice querySlice) {
```

Comment:
Can't this still be a const?

Reply:
Yes, it could still be const but as the VPackSlice is passed by value, making it const or non-const won't change the API. It only affects the const-ness of the `querySlice` variable inside `handleUseQuery`. As a `VPackSlice` does not have any mutators, it can always be passed as non-const copy.